7.4/config/php/xdebug.ini

```diff
@@ -6,3 +6,4 @@ xdebug.mode=debug
 ; xdebug.xdebug.remote_port is set at runtime: 9003 (default)
 xdebug.idekey=xdebug_session
 xdebug.max_nesting_level=256
+xdebug.port=9000
```

Comment:
Should be `xdebug.client_port`.

https://xdebug.org/docs/all_settings#client_port